--- a/src/decode/sondedump/CMakeLists.txt	2025-08-15 20:11:10.113957513 +0200
+++ b/src/decode/sondedump/CMakeLists.txt	2025-08-15 20:07:23.811639973 +0200
@@ -27,7 +27,7 @@
 	add_definitions(-D__global=__attribute__\(\(visibility\(\"default\"\)\)\))
 	set(CMAKE_C_STANDARD 99)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -pipe -Wall -Wextra -pedantic -Wimplicit-fallthrough")
-	set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2 -ftree-vectorize -flto")
+	set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2 -ftree-vectorize")
 	if (FULL_OPTIMIZE)
 		set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -march=native")
 	endif()
