--- a/src/decode/sondedump/CMakeLists.txt	2025-08-15 20:11:10.113957513 +0200
+++ b/src/decode/sondedump/CMakeLists.txt	2025-08-15 20:07:23.811639973 +0200
@@ -27,7 +27,7 @@
 	add_definitions(-D__global=__attribute__\(\(visibility\(\"default\"\)\)\))
 	set(CMAKE_C_STANDARD 99)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -pipe -Wall -Wextra -pedantic -Wimplicit-fallthrough")
-	set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2 -ftree-vectorize -flto")
+	set(CMAKE_C_FLAGS_RELEASE "-DNDEBUG -O2 -ftree-vectorize")
 	if (FULL_OPTIMIZE)
 		set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -march=native")
 	endif()
--- a/src/main.cpp	2025-08-15 20:11:07.333564468 +0200
+++ b/src/main.cpp	2025-08-15 01:38:46.307258866 +0200
@@ -54,7 +54,7 @@
 	bw = std::get<1>(supportedTypes[typeToSelect]);
 	vfo = sigpath::vfoManager.createVFO(name, ImGui::WaterfallVFO::REF_CENTER, 0, bw, bw, bw, bw, true);
 	vfo->setSnapInterval(SNAP_INTERVAL);
-	fmDemod.init(vfo->output, bw, bw/2.0f, false);
+	fmDemod.init(vfo->output, bw, bw/2.0f, false, false);
 
 	/* Resampler to 48kHz */
 	resampler.init(&fmDemod.out, bw, OUT_SAMPLE_RATE);
